src/Components/Scenes/SearchResults/CheckBoxButton/CheckBoxButton.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { Button } from '@material-ui/core';
+import { BrowserRouter as Route, Link } from 'react-router-dom';
```

Comment:
Linting fix - no need to import `Route`

Reply:
I think this is old one. 
Solved

Reply:
I think this is old version of my code. Removed Solved